src/sctools/test/test_metrics.py

```diff
@@ -52,78 +55,79 @@ def gene_metrics():
-    fragments_expected = int(_testing_knowledge['n_fragments'])
+    fragments_expected = int(_testing_knowledge_scalar['n_fragments'])
     assert fragments_expected == fragments_observed
 
 
 def test_metrics_file_identifies_correct_highest_expression_gene(gene_metrics):
     observed_max_gene = gene_metrics['n_reads'].idxmax()
-    expected_max_gene = _testing_knowledge['most_abundant']
+    expected_max_gene = _testing_knowledge_scalar['most_abundant']
     assert expected_max_gene == observed_max_gene
 
 
 def test_metrics_file_identifies_correct_highest_expression_count(gene_metrics):
     observed_max_gene_reads = gene_metrics['n_reads'].max()
-    expected_max_gene_reads = int(_testing_knowledge['most_abundant_gene_n_observations'])
+    expected_max_gene_reads = int(_testing_knowledge_scalar['most_abundant_gene_n_observations'])
     assert expected_max_gene_reads == observed_max_gene_reads
 
 
 def test_metrics_file_identifies_correct_number_perfect_barcodes(gene_metrics):
     observed_perfect_barcodes = gene_metrics['perfect_molecule_barcodes'].sum()
-    expected_perfect_barcodes = int(_testing_knowledge['perfect_molecule_barcodes'])
+    expected_perfect_barcodes = int(_testing_knowledge_scalar['perfect_molecule_barcodes'])
     assert observed_perfect_barcodes == expected_perfect_barcodes
 
 
 def test_reads_mapped_exonic(gene_metrics):
     observed = gene_metrics['reads_mapped_exonic'].sum()
-    expected = int(_testing_knowledge['reads_mapped_exonic'])
+    expected = int(_testing_knowledge_scalar['reads_mapped_exonic'])
     assert observed == expected
 
 
 # todo this is a zero-case in the current test data; need some intronic reads
 def test_reads_mapped_intronic(gene_metrics):
     observed = gene_metrics['reads_mapped_intronic'].sum()
-    expected = int(_testing_knowledge['reads_mapped_intronic'])
+    expected = int(_testing_knowledge_scalar['reads_mapped_intronic'])
     assert observed == expected
 
 
 # todo this is a zero-case in the current test data; need some utr reads
 def test_reads_mapped_utr(gene_metrics):
     observed = gene_metrics['reads_mapped_utr'].sum()
-    expected = int(_testing_knowledge['reads_mapped_utr'])
+    expected = int(_testing_knowledge_scalar['reads_mapped_utr'])
     assert observed == expected
 
 
 def test_reads_mapped_uniquely(gene_metrics):
     observed = gene_metrics['reads_mapped_uniquely'].sum()
-    expected = int(_testing_knowledge['reads_mapped_uniquely'])
+    expected = int(_testing_knowledge_scalar['reads_mapped_uniquely'])
     assert observed == expected
 
 
 def test_duplicate_records(gene_metrics):
     observed = gene_metrics['duplicate_reads'].sum()
-    expected = int(_testing_knowledge['duplicate_reads'])
+    expected = int(_testing_knowledge_scalar['duplicate_reads'])
     assert observed == expected
 
 
+    # todo this test currently fails because we believe our data is not deduped correctly, need to run on data run on latest optimus pipeline
```

Comment:
Lets change this comment to reflect that we're perplexed, not a failure of deduping.